boards/arm/colibri_imx7d_m4/Kconfig.defconfig

```diff
@@ -58,5 +58,21 @@ config I2C_4
 
 endif # I2C_IMX
 
+if PWM_IMX
+
+config PWM_1
+	def_bool y
+
+config PWM_2
+	def_bool n
```

Comment:
The `def_bool n` are redundant. I think all but PWM_1 should be removed. 

@ulfalizer Will you please confirm?